README.md

```diff
@@ -62,6 +62,7 @@ npm start
 |LOCAL_USERNAME|Username of a user for your instance|
 |LOCAL_PASSWORD|Password of a user for your instance|
 |PURGE_OLDER_THAN_DAYS|Posts must be older than this many days to be purged|
+|PURGE_PAGE_SIZE|Optional: Process this many posts at a time. Default: 100|
```

Comment:
Would suggest `PURGE_BATCH_SIZE` and a rename of other variable to reflect this

Reply:
agreed